.travis.yml

```diff
@@ -1,13 +1,20 @@
 language: node_js
+node_js:
+  - "6"
+  - "8"
+  - "10"
 
 sudo: false
+dist: trusty
 
-node_js:
-  - "6.0"
-  - "7.0"
+addons:
+  chrome: stable
```

Comment:
I had no idea you could do that 👏

Reply:
[Yup](https://docs.travis-ci.com/user/chrome), but I only knew from riding the [coattails of ember-cli](https://github.com/ember-cli/ember-cli/pull/7161).